app.yaml

```diff
@@ -0,0 +1,5 @@
+runtime: nodejs
```

Comment:
Does this come with any predefined environment variables?

If not, [setting `NODE_ENV` to `production`](http://expressjs.com/en/advanced/best-practice-performance.html#set-nodeenv-to-production) will make Express cache the view template